### PR TITLE
1906-03-26

### PR DESCRIPTION
**Pull request by hlb17**
deliverable 1

### CODE REVIEW

File: 1906-03-26.xml [hunk 1]
```diff
@@ -0,0 +1,1352 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="https://raw.githubusercontent.com/dig-eg-gaz/resources/master/out/egSchema.rnc" type="application/relax-ng-compact-syntax"?>
+<?xml-stylesheet type="text/xsl" href="dig-eg-gaz.xsl"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title level="m" type="main">Digital Egyptian Gazette</title>
+                <title level="m" type="sub">An encoded transcription</title>
+                <editor><!-- Hannah Bridges--></editor>
```
Comment (whanley) [inline 1906-03-26.xml]: remove comment tag around name

**Comment by whanley** [comment]
There are two different files in this pull request--"1906-03-26.xml" and "1906-03-26 copy.xml". Please make sure that your latest changes are in the file named "1906-03-26.xml", delete the file named "1906-03-26 copy.xml", and send a new pull request.

**Comment by whanley** [comment]
I still see two files--after you delete the copy file from your branch and send a pull request, you should see just one file under the "Files Changed" tab above.

**Comment by hlb17** [comment]
Dogs-
Dogs being poised by Cairo police

**Comment by whanley** [comment]
Submit your xpath query here: bit.ly/deg-xpath-query

Your double file problem still needs to be resolved. See previous comments, or ask me in class.

**Pull request closed without merging**